api.raml

```diff
@@ -928,3 +977,119 @@ schemas:
           429:
             description: |
               Too many failed attempts for this MFA record recently.
+/method:
+  post:
+    description: Create a new recovery method and send verification email.
+    body:
+      application/json:
+        example: |
+          {
+            "employee_id": "12345",
+            "value": "987654"
+          }
+    responses:
+      200:
+        description: Recovery method created and email sent.
+        body:
+          application/json:
+            schema: MethodResponse
+            example: |
+              {
+                "uid": "12345678901234567890123456789012",
+                "value": "email@example.org"
```

Comment:
What is the meaning/significance of the `value` in the MethodResponse?

Reply:
`value` is always an email, as of today. I'll make this more clear.